### PR TITLE
1902 clean up v1 redux

### PR DESCRIPTION
**Pull request by cosa65**
# Description

# Details
#### URL to issue
https://biomage.atlassian.net/browse/BIOMAGE-1902

#### Link to staging deployment URL (or set N/A)
https://ui-martinfosco-ui751.scp-staging.biomage.net/

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [ ] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [ ] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [ ] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [ ] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [ ] Started end-to-end tests on the latest commit.

### Documentation updates
- [ ] Relevant Github READMEs updated **or** no GitHub README updates required.
- [ ] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [ ] Photo of a cute animal attached to this PR.

### CODE REVIEW

File: src/__test__/redux/actions/experiments/__snapshots__/switchExperiment.test.js.snap [hunk 1]
```diff
@@ -100,7 +100,6 @@ Object {
       "createdDate": "0000-00-00T00:00:00.000Z",
       "description": "Mock experiment 0",
       "id": "testae48e318dab9a1bd0bexperiment-0",
-      "lastViewed": undefined,
```
Comment (cosa65) [inline src/__test__/redux/actions/experiments/__snapshots__/switchExperiment.test.js.snap:103]: This wasn't used anywhere in the ui, so I ended up removing it

File: src/__test__/redux/actions/projects/deleteProject.test.js [hunk 2]
```diff
@@ -96,6 +94,9 @@ describe('deleteProject action', () => {
   });
 
   it('Dispatches event correctly for one sample', async () => {
+    // const response = new Response(JSON.stringify({}));
+    // fetchMock.mockResolvedValueOnce(response);
+
```
Comment (ivababukova) [inline src/__test__/redux/actions/projects/deleteProject.test.js]: remove this

File: src/pages/data-management/index.jsx [hunk 3]
```diff
@@ -46,7 +46,7 @@ const DataManagementPage = () => {
     dispatch(loadBackendStatus(experimentId));
   };
 
-  const areSamplesLoaded = () => !activeProject.samples.length
+  const sampleAreLoaded = () => !activeProject.samples.length
```
Comment (ivababukova) [inline src/pages/data-management/index.jsx]: add `s` after sample

File: src/redux/actions/experiments/createExperiment.js [hunk 4]
```diff
@@ -7,52 +7,43 @@ import {
...
-        body: JSON.stringify(experimentToSend),
+        body: JSON.stringify(newExperimentProperties),
       },
     );
 
     dispatch({
       type: EXPERIMENTS_CREATED,
       payload: {
-        experiment: newExperiment,
+        experiment: { createdDate, ...newExperimentProperties },
```
Comment (ivababukova) [inline src/redux/actions/experiments/createExperiment.js:50]: remember to take a look at using the createdDate from the api as part of another set of PRs

**Review by ivababukova — APPROVED** [review]

**Pull request merged**